glossary/repair.md

```diff
@@ -0,0 +1,8 @@
+type of [achievable](https://github.com/newatoms/guides/blob/ready/glossary/achievable.md), where you repair something which was broken, i.e. not working the way it should.
```

Comment:
I would suggest:
"A repair is a type of achievable for which you repair something..."